### ISSUE TITLE
Enum was gone

### ISSUE THREAD

**Issue by tokuhirom** [zostay/p6-Hash-MultiValue#1]
https://github.com/rakudo/rakudo/commit/2232157eb35ddbfafdd26f3e57787fd5e58229fd

In this commit,  @niner removed Enum from rakudo. But Hash::MultiValue still use Enum https://github.com/zostay/p6-Hash-MultiValue/blob/master/lib/Hash/MultiValue.pm6#L305

As a result, t/basic.t doesn't work with git HEAD.

```
candace➜  p6-Hash-MultiValue : master ✔ : ᐅ  perl6-m -Ilib t/basic.t
===SORRY!===
Function Enum needs parens to avoid gobbling block
at /Users/tokuhirom/dev/p6-Hash-MultiValue/lib/Hash/MultiValue.pm6:308
------>         }⏏<EOL>
  from t/basic.t:6
Missing block (apparently taken by 'Enum')
at /Users/tokuhirom/dev/p6-Hash-MultiValue/lib/Hash/MultiValue.pm6:309
------>         }⏏<EOL>
  from t/basic.t:6
```

**Comment by zostay** [comment]
Thanks for the report. That change has a number of important consequences and it requires a full refactor of this and probably [KnottyPair](https://github.com/zostay/perl6-KnottyPair) too. The good news is that the new change makes hashes and pairs much nicer to work with. @niner++

**Comment by zostay** [comment]
Ack, I'd forgotten this hadn't been ported to use KnottyPair yet, which is good because I decided to just go ahead and obsolete that module since it does not provide much of use now. I hope to have this fixed tonight or soon.

**Comment by zostay** [comment]
All fixed in 118a42aaff8aec0e18b513b4a12be9e6a46c2bb5.
